packages/frontend/src/redux/slices/tokenFiatValues/index.js

```diff
@@ -23,31 +24,18 @@ const tokenFiatValuesSlice = createSlice({
         name: SLICE_NAME,
         initialState,
         extraReducers: ((builder) => {
-            builder.addCase(fetchTokenFiatValues.pending, (state, action) => {
-                state.loading = true;
-                state.error = initialErrorState;
-            });
-
             builder.addCase(fetchTokenFiatValues.fulfilled, (state, action) => {
                 // Payload of .fulfilled is in the same shape as the store; just merge it!
                 // { near: { usd: x, lastUpdatedTimestamp: 1212312321, ... }
 
-                state.loading = false;
-                state.error = initialErrorState;
-
                 // Using merge instead of `assign()` so in the future we don't blow away previously loaded token
                 // prices when we load new ones with different token names
                 merge(state.tokens, action.payload);
             });
-
-            builder.addCase(fetchTokenFiatValues.rejected, (state, { error }) => {
-                state.loading = false;
-
-                // TODO: Localize this?
-                state.error = {
-                    message: error?.message || 'An error was encountered.',
-                    code: error?.code
-                };
+            handleAsyncThunkStatus({
+                asyncThunk: `${SLICE_NAME}/fetchTokenFiatValues`,
+                buildStatusPath: () => '',
```

Comment:
Should this return `[]` so that it doesn't add an empty string to the array when `buildStatusPath` gets called?

Reply:
The result will be the same, but an empty array has much more sense. Done.